nf_core/pipeline-template/{{cookiecutter.name_noslash}}/docs/usage.md

```diff
@@ -242,7 +241,7 @@ Use to set a top-limit for the default CPU requirement for each process.
 Should be a string in the format integer-unit. eg. `--max_cpus 1`
 
 ### `--plaintext_email`
-Set to receive plain-text e-mails instead of HTML formatted.
+Set to receive plain-text e-mails instead of HTML formatted.  
```

Comment:
Added whitespace? I think this can be reverted

Reply:
Should be fixed now in latest commit.